Common/Securities/SecurityPortfolioModel.cs

```diff
@@ -49,19 +49,9 @@ public virtual void ProcessFill(SecurityPortfolioManager portfolio, Security sec
             try
             {
                 // apply sales value to holdings in the account currency
-                if (security.Type == SecurityType.Future || security.Type == SecurityType.Cfd)
-                {
-                    // for futures/CFDs, we measure volume of sales, not notionals
```

Comment:
For future readers; standardized for all securities even though this is including notional. The idea is all other securities with leverage still count as volume, its still risk taken and capital deployed in the transaction so should still count towards volume.